src/client/components/SupplierEditor/SupplierEditorForm.react.js

```diff
@@ -169,56 +170,61 @@ class SupplierEditorForm extends Component {
-              />
-            )
-          })}
-
-          { this.renderField({ fieldName: 'taxIdentificationNo' }) }
-          { this.renderField({ fieldName: 'vatIdentificationNo' }) }
-          { this.renderField({ fieldName: 'globalLocationNo' }) }
-          { this.renderField({ fieldName: 'dunsNo' }) }
-
-          <div className='supplier-form-submit'>
-            <div className='text-right form-submit'>
-              <button className="btn btn-primary" onClick={ this.handleUpdate }>
-                { i18n.getMessage('SupplierEditor.ButtonLabel.save') }
-              </button>
+        <div>
+          <h4 className="tab-description">
+            { i18n.getMessage(`SupplierEditor.Description.viewSupplierOrChooseAnother`) }
+          </h4>
+          <form className="form-horizontal">
+            { this.renderField({ fieldName: 'supplierName' }) }
+            { this.renderField({ fieldName: 'homePage' }) }
+            { this.renderField({
+              fieldName: 'foundedOn',
+              component: (
+                <DateInput
+                  className="form-control"
+                  locale={i18n.locale}
+                  dateFormat={dateTimePattern}
+                  value={foundedOn}
+                  onChange={this.handleChange.bind(this, 'foundedOn')}
+                  onBlur={this.handleBlur.bind(this, 'foundedOn')}
+                  variants={[]}
+                />
+              )
+            }) }
+
+            { this.renderField({ fieldName: 'legalForm' }) }
+            { this.renderField({ fieldName: 'commercialRegisterNo' }) }
+            { this.renderField({ fieldName: 'cityOfRegistration' }) }
+
+            { this.renderField({
+              fieldName: 'countryOfRegistration',
+              component: (
+                <CountryField
+                  actionUrl={this.props.actionUrl}
+                  value={this.state.supplier['countryOfRegistration']}
+                  onChange={this.handleChange.bind(this, 'countryOfRegistration')}
+                  onBlur={this.handleBlur.bind(this, 'countryOfRegistration')}
+                />
+              )
+            })}
+
+            { this.renderField({ fieldName: 'taxIdentificationNo' }) }
+            { this.renderField({ fieldName: 'vatIdentificationNo', marked: true }) }
+            { this.renderField({ fieldName: 'globalLocationNo', marked: true }) }
+            { this.renderField({ fieldName: 'dunsNo', marked: true }) }
+
+            <div className='supplier-form-submit'>
+              <div className='text-right form-submit'>
+                <button className="btn btn-primary" onClick={ this.handleUpdate }>
+                  { i18n.getMessage('SupplierEditor.ButtonLabel.save') }
+                </button>
+              </div>
             </div>
-          </div>
-        </form>
+          </form>
+        </div>
+        <div className="text-right col-md-4">
```

Comment:
Remove classNames - does not look nice in UI
![bildschirmfoto 2017-09-04 um 16 04 58](https://user-images.githubusercontent.com/1188617/30029849-31d2be00-918b-11e7-86ff-ed5242f627a2.png)